modules/user_accounts/jsx/permissions.js

```diff
@@ -0,0 +1,337 @@
+    );
+  }
+});
+
+var Permission = React.createClass({
+  handleCheck: function(e) {
+    this.props.onChange(this.props.id);
+  },
+  render: function() {
+    var textStyle = {};
+    if (this.props.disabled) {
+      textStyle = {
+        color: '#ccc',
+        cursor: 'not-allowed'
+      };
+    }
+    if (this.props.highlight) {
+      textStyle = {
+        color: 'lightblue'
+      };
+    }
+    return (
+      <div className="checkbox permission">
+        <label style={textStyle}>
+          <input
+            name={"permID[" + this.props.id + "]"}
+            type="checkbox"
+            value={this.props.id}
+            checked={this.props.checked}
+            onChange={this.handleCheck}
+            disabled={this.props.disabled}
+          />
+          {this.props.name}
+        </label>
+      </div>
+    );
+  }
+});
+
+var PermissionList = React.createClass({
+  render: function() {
+    var half = Math.ceil(this.props.data.length / 2);
+    var firstHalf = this.props.data.slice(0, half);
+    var secondHalf = this.props.data.slice(half, this.props.data.length);
+    var permissionNodesCol1 = firstHalf.map(function(permission) {
+      return (
+        <Permission
+          key={permission.id}
+          id={permission.id}
+          name={permission.name}
+          highlight={permission.highlight}
+          checked={permission.checked}
+          onChange={this.props.onChange}
+          disabled={permission.disabled} />
+      );
+    }, this);
+    var permissionNodesCol2 = secondHalf.map(function(permission) {
+      return (
+        <Permission
+          key={permission.id}
+          id={permission.id}
+          name={permission.name}
+          highlight={permission.highlight}
+          checked={permission.checked}
+          onChange={this.props.onChange}
+          disabled={permission.disabled} />
+      );
+    }, this);
+    return (
+      <div className="permissionList">
+        <div className="col-md-6">
+          {permissionNodesCol1}
+        </div>
+        <div className="col-md-6">
+          {permissionNodesCol2}
+        </div>
+      </div>
+    );
+  }
+});
+
+var Access = React.createClass({
+  getInitialState: function() {
+    return {
+      roles: [],
+      permissions: [],
+      rolesUsed: true
+    };
+  },
+  loadDataFromServer: function() {
+    $.ajax({
+      url: this.props.dataURL,
+      data: {identifier: this.props.identifier},
+      dataType: 'json',
+      cache: false,
+      success: function(data) {
+        this.setState({
+          roles: data.roles,
+          permissions: data.permissions
+        });
+        if (!this.state.roles.length) {
+          this.setState({
+            rolesUsed: false
+          });
+        }
+      }.bind(this),
+      error: function(xhr, status, err) {
+        console.error(this.props.dataURL, status, err.toString());
+      }.bind(this)
+    });
+  },
+  componentDidMount: function() {
+    this.loadDataFromServer();
```

Comment:
I've added this, but it's still gonna try to insert the component on user accounts pages that are not edit_user, resulting in this kind of error:
<img width="553" alt="screen shot 2017-03-29 at 11 27 45 am" src="https://cloud.githubusercontent.com/assets/5473436/24448840/a5924fd8-1475-11e7-96b4-354b3d13cbed.png">

Is there an elegant way to avoid this?